packages/cdk-cache-invalidator/README.md

```diff
@@ -1 +1 @@
-# @codedazur/cdk-static-site
```

Comment:
We have some naming conflicts I think we use `@codedazur/cdk-cache-invalidator` and `@codedazur/cdk-rest-api` and `@codedazur/cdk-api`. Let's pick one 😬